src/index.html

```diff
@@ -1,17 +1,153 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
+    <title>Hello</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
   </body>
+  <main>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post">
+      <fieldset class="form-field">
+        <legend>Personal information</legend>
+        <label class="form-wrap">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form-wrap">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form-wrap">
+          How old are You?
+          <input
+            type="number"
+            name="howOldAreYou"
+            value="12"
+            max="100"
+            min="1"
+          >
+        </label>
+        <label class="form-wrap">
+          Full date of birth:
+          <input type="date" name="date">
+        </label>
+        <label>
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="accept_agreement"
+            value="accept"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-field">
+        <legend>Registration
+        </legend>
```

Comment:
those should be either in one line or each on a separate line
```suggestion
        <legend>
          Registration
        </legend>
```